examples/example_EEG.py

```diff
@@ -0,0 +1,169 @@
+    synapse = LFPy.Synapse(cell, **synapse_params)
+    synapse.set_spike_times(np.array([5.]))
+
+    cell.simulate(rec_imem=True,
+                  rec_current_dipole_moment=True)
+
+    # compute dipole
+    P = cell.current_dipole_moment
+
+    somapos = np.array([0., 0., 77500])
+    # cell.set_pos(x=somapos[0], y=somapos[1], z=somapos[2])
+    r_soma_syns = [cell.get_intersegment_vector(idx0=0, idx1=i) for i in cell.synidx]
+    r_mid = np.average(r_soma_syns, axis=0)
+    r_mid = somapos + r_mid/2.
+
+    eeg_coords_top = np.array([[0., 0., radii[3] - rad_tol]])
+    four_sphere_top = LFPy.FourSphereVolumeConductor(radii, sigmas, eeg_coords_top, r_mid)
+    pot_db_4s_top = four_sphere_top.calc_potential(P)
+    eeg_top = np.array(pot_db_4s_top) * 1e9
+
+    #measurement points
+    # for nice plot use theta_step = 1 and phi_step = 1. NB: Long computation time.
+    theta_step = 5
+    phi_step = 5
+    theta, phi_angle = np.mgrid[0.:180.:theta_step, 0.:360.+phi_step:phi_step]
+
+    num_theta = theta.shape[0]
+    num_phi = theta.shape[1]
+    theta = theta.flatten()
+    phi_angle = phi_angle.flatten()
+
+    theta_r = np.deg2rad(theta)
+    phi_angle_r = np.deg2rad(phi_angle)
+
+    x_eeg = (radii[3] - rad_tol) * np.sin(theta_r) * np.cos(phi_angle_r)
+    y_eeg = (radii[3] - rad_tol) * np.sin(theta_r) * np.sin(phi_angle_r)
+    z_eeg = (radii[3] - rad_tol) * np.cos(theta_r)
+    eeg_coords = np.vstack((x_eeg, y_eeg, z_eeg)).T
+
+    # potential in 4S with db
+    time_max = np.argmax(np.linalg.norm(P, axis=1))
+    p = P[time_max, None]
+    four_sphere = LFPy.FourSphereVolumeConductor(radii, sigmas, eeg_coords, r_mid)
+    pot_db_4s = four_sphere.calc_potential(p)
+    eeg = pot_db_4s.reshape(num_theta, num_phi)*1e9# from mV to pV
+
+    P_mag = np.sqrt(P[:, 0]**2 + P[:, 1]**2 + P[:, 2]**2)
+
+    plt.close('all')
+    fig = plt.figure(figsize=[5, 8])
+    fig.subplots_adjust(left=0.15, hspace=0.5, right=0.98, top=0.95)
+
+    morph_ax = fig.add_subplot(321, aspect=1, frameon=False,
+                               title="Cell and synapse",
+                               xticks=[], yticks=[])
+    plot_cell_to_ax(cell, morph_ax, cell.synidx)
+
+    plot_EEG_sphere(fig, eeg, x_eeg, y_eeg, z_eeg)
+
+    ax_p = fig.add_subplot(312, title="Current dipole moment", ylabel="nA$\cdot\mu$m")
+    # ax_p.plot(cell.tvec, P_mag, label="|P|")
+    ax_p.plot(cell.tvec, P[:, 0], label="P$_x$")
+    ax_p.plot(cell.tvec, P[:, 1], label="P$_y$")
+    ax_p.plot(cell.tvec, P[:, 2], label="P$_z$")
+    ax_p.axvline(cell.tvec[time_max], c='gray', ls='--')
+    ax_p.legend(frameon=False, ncol=4, bbox_to_anchor=[1, -0.1])
+
+    ax_eeg = fig.add_subplot(313, title="EEG at top", xlabel="Time (ms)",
+                             ylabel='pV')
+    ax_eeg.plot(cell.tvec, eeg_top[0, :], 'k')
+    # ax_eeg.plot(cell.tvec, P[:, 2], c='g', ls='--', label="z")
+
+    ax_eeg.axvline(cell.tvec[time_max], c='gray', ls='--')
+
+    plt.savefig('example_EEG.pdf')
```

Comment:
Add a `plt.show()` before printing. Showing plots is not default with the macos backend.